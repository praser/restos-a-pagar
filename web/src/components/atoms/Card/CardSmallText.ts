import styled from 'styled-components';

const CardSmallText = styled.p`
  font-size: 12px;
  color: darkgray;
`;

export default CardSmallText;
