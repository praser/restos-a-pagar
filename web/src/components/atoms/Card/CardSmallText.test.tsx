import React from 'react';

import renderer from 'react-test-renderer';

import CardSmallText from './CardSmallText';

describe('CardSmallText atom component', () => {
  it('is expected to render the component correctly', () => {
    const cardSmallText = renderer.create(<CardSmallText />).toJSON();
    expect(cardSmallText).toMatchSnapshot();
  });
});
