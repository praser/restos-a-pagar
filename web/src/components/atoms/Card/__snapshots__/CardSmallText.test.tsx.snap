// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardSmallText atom component is expected to render the component correctly 1`] = `
<p
  className="sc-bdfBwQ hQsQJW"
/>
`;
